templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% block content %}
+</h1>Delete a Manufacturer</h1>
+
+<p>Are you sure you want to delete the manufacturer: {{ manufacturer }} ?</p>
```

Comment:
Consider adding a `Cancel` button that leads back to the previous page, as per the checklist requirement: 'Add `Cancel` button apart from `Delete` one. The `Cancel` button will lead to the previous page the user was on.'